vispy/scene/cameras/perspective.py

```diff
@@ -30,12 +30,12 @@ class PerspectiveCamera(BaseCamera):
     _state_props = ('scale_factor', 'center', 'fov')
 
     def __init__(self, fov=60.0, scale_factor=None, center=None, **kwargs):
+        kwargs['fov'] = fov
         super(PerspectiveCamera, self).__init__(**kwargs)
```

Comment:
```suggestion
        super(PerspectiveCamera, self).__init__(fov=fov, **kwargs)
```

Reply:
This does make more sense to me. I'm a little worried that setting `self.fov` this early in the `__init__` is going to be a problem for some cameras and may not be fully tested. My main fear is that `view_changed` is called in the `fov` setter and *then* the rest of the perspective base class `__init__` is run.